src/netconfig.rs

```diff
@@ -8,6 +8,7 @@ pub struct Netconfig {
     pub static_dns_servers: Option<Vec<String>>,
     pub static_dns_searchlist: Option<Vec<String>>,
     pub dns_policy: Vec<String>,
+    pub warnings: Vec<String>,
```

Comment:
The other warnings we have in `interface.rs` are of type `Vec<anyhow::Error>` don't you think this makes also more sense here?

Reply:
:thinking: I guess you are right